pyproject.toml

```
@@ -28,7 +28,8 @@ dependencies = [
   "tqdm",
   "python-dotenv", 
   "scikit-image", 
-  "opencv-python"
+  "opencv-python",
+  "stamina"
```

```suggestion
  "stamina",
```

TOML supports trailing commas, which helps reduce the diff/blame noise if you need to append something in the future.